ROADMAP.md

```diff
@@ -0,0 +1,18 @@
+# Q3 2020 Roadmap
+
+Each quarter, the team will highlight areas of focus for our work and upcoming research.
+Each release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. This upcoming calendar quarter (Jul-Sep 2020) we will be prioritizing the following areas of work:
+
+## Currently In Progress
+### v2.x Release
+With the release of v2.0.0 we will continue to make improvements to the SDK to ease the upgrade path for provider developers who have not yet migrated and improve the functionality for those that have.
+ 
+## Planned
+### Protocol-native provider development packages
+As part of a project to enhance the Terraform Provider developer experience redesign the Terraform Plugin SDK, we will be releasing packages to interact with the Terraform protocol directly, with little to no abstraction. Future packages will build on these to offer abstractions and frameworks similar to the current SDK’s experience. Providers may wish to make use of these packages for more complex resources that need advanced behavior not supported natively in our current SDK.
```

Comment:
> enhance the Terraform Provider developer experience redesign the Terraform Plugin SDK

That doesn't make much sense

Reply:
oh weird, looks like I somehow copied over a comment. fixed, thanks for catching @paddycarver.